simpleprophet/README.md

```diff
@@ -0,0 +1,35 @@
+# simpleprophet KPI forecasting models and tools
+
+This directory contains the simpleprophet forecast models and accompanying tools.  The simpleprophet models prioritize simplicity and stability, adding complexity and "bendiness" only when it significantly improves model performance on a holdback period.  More documentation to come.
+
+## data.py
+
+Tools for getting metric actual or forecast data from BigQuery.
```

Comment:
These would be great as module-level docstrings either instead of or in addition to the docs here. Example below.

Reply:
Merging to unblock you, but will make those changes.